src/components/featured-products-section/featured-products-section.tsx

```diff
@@ -34,7 +33,7 @@ export const FeaturedProductsSection = (props: FeaturedProductsSectionProps) =>
                           <ProductLink key={product._id} productSlug={product.slug!}>
                               <ProductCard
                                   name={product.name!}
-                                  imageUrl={product.media?.mainMedia?.image?.url}
+                                  imageUrl={getProductImageUrl(product, 700, 700)}
```

Comment:
Consider using named args here to make values meaningful

Reply:
Changed to:
```ts
getProductImageUrl(product, { minHeight: 700 })
```